docs/src/man/primitives.jl

```diff
@@ -4,7 +4,8 @@ using SolidStateDetectors
 import SolidStateDetectors.ConstructiveSolidGeometry as CSG
 using SolidStateDetectors.ConstructiveSolidGeometry: ClosedPrimitive
 using SolidStateDetectors.ConstructiveSolidGeometry: Box
-using Plots
+using Plots; gr()
+Plots.reset_defaults()
```

Comment:
What does this do?

Reply:
No it didn't...

Reply:
30e6216 solved it. I removed the `Plots.reset_defaults` again (force push).

Reply:
well, `Plots.reset_defaults()` still seems to be in that file..

Reply:
Ah, right. It was in `primitives.jl` as well... We can clean this up later.